tests/queries/0_stateless/02417_opentelemetry_insert_on_distributed_table.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+# Tags: distributed
+
+set -ue
+
+unset CLICKHOUSE_LOG_COMMENT
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+
+${CLICKHOUSE_CLIENT} -nq "
+SET distributed_ddl_output_mode = 'none';
+
+SYSTEM FLUSH LOGS ON CLUSTER test_cluster_two_shards;
+TRUNCATE TABLE IF EXISTS system.opentelemetry_span_log ON CLUSTER test_cluster_two_shards;
+
+DROP TABLE IF EXISTS default.dist_opentelemetry ON CLUSTER test_cluster_two_shards;
+DROP TABLE IF EXISTS default.local_opentelemetry ON CLUSTER test_cluster_two_shards;
+
+CREATE TABLE default.dist_opentelemetry  ON CLUSTER test_cluster_two_shards (key UInt64) Engine=Distributed('test_cluster_two_shards', default, local_opentelemetry, key % 2);
+CREATE TABLE default.local_opentelemetry ON CLUSTER test_cluster_two_shards (key UInt64) Engine=MergeTree ORDER BY key;
+"
+
+#
+# INSERT ASYNC test
+# Do test with opentelemetry enabled
+#
+${CLICKHOUSE_CLIENT} -nq "
+-- Make sure it's async
+SET insert_distributed_sync=0;
+INSERT INTO default.dist_opentelemetry SETTINGS opentelemetry_start_trace_probability=1 VALUES(1),(2);
```

Comment:
Is it possible to pass the setting via `SETTINGS` suffix to `INSERT INTO`? (that would resetting (l. 76) obsolete ... not that in case of a spurious error that reset might not happen and we'll affect subsequent tests)

(same for the other `SET` statements in this file)